dotCMS/src/main/java/com/dotcms/util/pagination/CategoriesPaginator.java

```diff
@@ -0,0 +1,59 @@
+package com.dotcms.util.pagination;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+import com.dotcms.repackage.com.google.common.annotations.VisibleForTesting;
+import com.dotcms.repackage.org.apache.commons.lang.StringUtils;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.RoleAPI;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.business.RoleAPI'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)